docs/attribution.md

```diff
@@ -1,19 +1,14 @@
 # Attribution
 
-
-Attribution is required for many of our data providers. Example language is provided below, but you are responsible for researching each project to follow their license terms. More details are available on the [Data Sources](data-sources.md) page.
+Attribution is required for using Mapzen's hosted vector tile service as well as for many of our data providers. Example language is provided below, but you are responsible for researching each project to follow their license terms. More details are available on the [Data Sources](data-sources.md) and the [Mapzen rights](https://mapzen.com/rights) pages.
 
 **Required attribution:**
```

Comment:
ahhh, I misunderstood. Double checking the wording on Terrain and adding a commit.

also yay Natural Earth!